### DISCUSSION TITLE
Fix Problem Params 'module' and 'action' are required parameters

### DISCUSSION THREAD

**Discussion by kasrakhamesi** [blockscout/blockscout#10062] [Q&A]
I get error : 
 Params 'module' and 'action' are required parameters
 
in http://localhost/api/v1/counters and http://localhost/api/v1/lines

all of project running successfully but when I want open http://localhost/stats in frontend, I get top error

I using docker-compose -f geth-clique-consensus.yml up -d for run blockscout project

please help me to fix that

**Comment by sepbin** [comment]
me too...

**Reply by GimluCom** [reply]
By default it should be http://localhost:8080/api/v1/counters
Ensure that you have set `NEXT_PUBLIC_STATS_API_HOST=http://localhost:8080` in the [`envs/common-frontend.env`](https://github.com/blockscout/blockscout/blob/v6.10.2/docker-compose/envs/common-frontend.env#L3) file.

**Comment by Hhsjkd829bx** [comment]
Me to
